packages/edit-post/src/components/browser-url/index.js

```diff
@@ -98,7 +99,7 @@ export class BrowserURL extends Component {
 }
 
 export default withSelect( ( select ) => {
-	const { getCurrentPost, isSavingPost } = select( 'core/editor' );
+	const { getCurrentPost, isSavingPost } = select( editorStore );
```

Comment:
Did we mean to do `editorStore.name` in this and all the following changes?

Reply:
:thinking: does `select` here expect a `string` instead of store? I know it does in https://github.com/WordPress/gutenberg/pull/32195/files#diff-ec3b73f716ad961aaf2ea0d6382631f600e7d6dfc60905aa0e3775c102156fa2, but I don't believe that is the case for the rest of them (please correct me if I'm wrong)

Reply:
A store definition should also work 👍  Just wanted to verify that we're intentionally changing from store name to a store definition.